go.mod

```diff
@@ -125,3 +125,5 @@ require (
 go 1.13
 
 replace github.com/docker/distribution v2.7.1+incompatible => github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible
+
+replace github.com/jessevdk/go-flags => github.com/vito/go-flags v1.4.1-0.20200421142706-6149d40139cb
```

Comment:
this particular version of vito/go-flags lacks an important feature: while it is nice that it gives errors on parsing env vars, it doesn't actually identify _which_ flag caused the error. this is pretty tough to sift through.

Reply:
vito/go-flags#1 addresses this, let's remember to update the dependency here.

Reply:
6de27d2aa803323239eb77cc7216bd63ecbe2975